integration/tests/cook/test_basic.py

```diff
@@ -54,7 +54,7 @@ def test_no_cook_executor_on_subsequent_instances(self):
             pass
```

Comment:
The `self.assertEqual` will be caught and ignored by this try/accept block if they fail.